manifest.json

```diff
@@ -1788,6 +1788,32 @@
                 }
             }
         },
+        "epicEaston197.bspimporter": {
+            "name": "BSPImporter",
+            "description": "Import bsp files into Neos",
+            "category": "Asset Importing Tweaks",
+            "website": "https://github.com/epicEaston197/BSPImporter",
+            "sourceLocation": "https://github.com/epicEaston197/BSPImporter",
+            "authors": {
+                "epicEaston197": {
+                    "url": "https://github.com/epicEaston197"
+                },
+                "amber": {
+                    "url": "https://github.com/kawaiiamber"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/epicEaston197/BSPImporter/releases/tag/1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/epicEaston197/BSPImporter/releases/1.0.0/download/BSPImporter.dll",
```

Comment:
This is a 404 for me.

Reply:
Oh the whole DLL file has disappeared, didn't double check it as I had previously already checked it and matched the hash. Original URL wouldn't work either now.

Reply:
i had also downloaded a copy of this artifact to verify its hash, here it is. (github forced me to zip the file)
[BSPImporter.zip](https://github.com/neos-modding-group/neos-mod-manifest/files/8666921/BSPImporter.zip)

Reply:
@epicEaston197 should probably reply as to if there's any plans to put the file back up?
Since as things stand now, I feel like this issue should be closed in a few days if there's no more information about this as the download links are now unavailable.
Since even if someone was to create a fork for distributing the files, that'd be a separate PR I feel like.

Reply:
based on a discussion in the nml discord Easton said the mod doesn't even work